recipes.js

```diff
@@ -2,9 +2,67 @@ const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
 const data = require('./data.js');
 
+const Recipe = mongoose.Schema 
+
 mongoose.connect('mongodb://localhost/recipeApp')
   .then(() => {
     console.log('Connected to Mongo!');
   }).catch(err => {
     console.error('Error connecting to mongo', err);
   });
+
+const recipeSchema= new Schema ({
+  title: {
+    type: String,
+    required: true,
+    unique: true},
+  level: {
+    type: String,
+    enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef', ] },
+  ingredients: {
+    type: Array},
+  cuisine: {
+    type: String,
+    required: true},
+  dishType: {
+    type: String,
+    enum: ['Breakfast','Dish','Snack','Drink','Dessert','Other']},
+  image: {
+    type: String, 
+    default: `https://images.media-allrecipes.com/images/75131.jpg.`},
+  duration: {
+    type: Number,
+    min: 0},
+  creator: {
+    type: String},
+  created: {
+    type: Date,
+    default: Date.now}
+
+});
+
+Recipe.Create({
+  title: 'Breakfast Burrito',
+  level: 'UltraPro Chef',
+  cuisine: 'BrexMex'
+})
+
+  Recipe.then (recipeDoc => {
+    console.log("RECIPE CREATE WORKED!! :)", recipeDoc);
+})
+  Recipe.catch (err => {
+    console.log("RECIPE CREATE failed :(", err)
+});
```

Comment:
.then and .catch are useful to know if your database query worked or failes, you cain chain them directly to the .create(...) method, you can't call them on Recipe directly, because it is not a function : 
```js
Recipe.create({
  title: 'Breakfast Burrito',
  level: 'UltraPro Chef',
  cuisine: 'BrexMex'
}).then (recipeDoc => {
    console.log("RECIPE CREATE WORKED!! :)", recipeDoc);
})
  .catch (err => {
    console.log("RECIPE CREATE failed :(", err)
});
```